modules/kmm-uninstalling-kmm.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="uninstalling-kmmo_{context}"]
+= Uninstalling the Kernel Module Management Operator
+
+You can use one of the following procedures to uninstall the Kernel Module Management (KMM) Operator:
+
+ * Use this procedure if KMM was installed from the Red Hat catalog.
++
+.Procedure
++
+. Either use the OpenShift console under *Operators* --> *Installed Operators* (preferred), or delete the ``Subscription`` resource in the KMM namespace.
+
+
+* Use this command if KMM was installed using the OpenShift CLI.
++
+.Procedure
++
+[source,terminal]
+----
+oc delete -k https://github.com/rh-ecosystem-edge/kernel-module-management/config/default
```

Comment:
I think we should add a not there explaining that that method with delete `Module` instances in the cluster.

Reply:
Added Note: "Using this command deletes the ``Module`` CRD and all ``Module`` instances in the cluster."